app/mailers/planning_application_mailer.rb

```diff
@@ -41,14 +41,13 @@ def invalidation_notice_mail(planning_application, host)
     )
   end
 
-  def receipt_notice_mail(planning_application, host, user)
-    @host = host
+  def receipt_notice_mail(planning_application, email)
     @planning_application = planning_application
 
     view_mail(
       NOTIFY_TEMPLATE_ID,
-      subject: "We have received your application",
-      to: user,
+      subject: "Lawful Development Certificate application received",
```

Comment:
Out of scope but possibly could eventually move this stuff to the locale so we can manage all subjects there

Reply:
Good call!